tutorials/assets_pipeline/escn_exporter/index.rst

```diff
@@ -3,48 +3,12 @@
 Blender ESCN exporter
 =====================
 
-.. note:: This chapter relates to the Blender plugin called "Godot Blender Exporter",
-          which can be downloaded here: https://github.com/godotengine/godot-blender-exporter
+To export from Blender to Godot 4.x, use one of the 
+:ref:`available 3D formats <doc_importing_3d_scenes_available_formats>`.
 
-This plugin can be used to export Blender scenes in a Godot-specific scene format
-called ESCN, which is similar to TSCN (text format) but will be imported as binary
-SCN for performance.
+The plugin `Godot Blender Exporter <https://github.com/godotengine/godot-blender-exporter>`__ 
+is not maintained or supported in Godot 4.x. While not officially supported, the plugin may 
+partially work for some Godot and Blender versions, particularly before Blender version 4.0.
```

Comment:
If we don't even want to suggest this much partial functionality, it's fine by me.